plugins/modules/nsxt_upgrade_run.py

```diff
@@ -72,6 +71,8 @@ def get_upgrade_status(module, manager_url, mgr_username, mgr_password, validate
   '''
   no_of_checks = 0
   while True:
+    endpoint = "/upgrade/upgrade-unit-groups?sync=true"
```

Comment:
Bug ID Fixed - 2687520 - upgrade module 'nsxt_upgrade_run' gets stuck even after MP upgrade completion is 100 percent. and edge upgrade status is cleared from from 100% to 0.0

Added a sync call before fetching the results of upgrade.